stable/openldap/README.md

```diff
@@ -23,27 +23,34 @@ We use the docker images provided by https://github.com/osixia/docker-openldap.
+| `image.pullPolicy`                 | Container pull policy                                                     | `IfNotPresent`    |
+| `extraLabels`                      | Labels to add to the Resources                                            | `{}`              |
+| `existingSecret`                   | Use an existing secret for admin and config user passwords                | `""`              |
+| `service.annotations`              | Annotations to add to the service                                         | `{}`              |
+| `service.clusterIP`                | IP address to assign to the service                                       | `""`              |
+| `service.externalIPs`              | Service external IP addresses                                             | `[]`              |
+| `service.ldapPort`                 | External service port for LDAP                                            | `389`             |
+| `service.loadBalancerIP`           | IP address to assign to load balancer (if supported)                      | `""`              |
+| `service.loadBalancerSourceRanges` | List of IP CIDRs allowed access to load balancer (if supported)           | `[]`              |
+| `service.sslLdapPort`              | External service port for SSL+LDAP                                        | `636`             |
+| `service.type`                     | Service type                                                              | `ClusterIP`       |
+| `env`                              | List of key value pairs as env variables to be sent to the docker image. See https://github.com/osixia/docker-openldap for available ones | `[see values.yaml]`  |
+| `adminPassword`                    | Password for admin user. Unset to auto-generate the password              | None              |
+| `configPassword`                   | Password for config user. Unset to auto-generate the password             | None              |
+| `customLdifFiles`                  | Custom ldif files to seed the LDAP server. List of filename -> data pairs | None              |
+| `persistence.enabled`              | Whether to use PersistentVolumes or not                                   | `false`           |
+| `persistence.storageClass`         | Storage class for PersistentVolumes.                                      | `<unset>`         |
+| `persistence.accessMode`           | Access mode for PersistentVolumes                                         | `ReadWriteOnce`   |
+| `persistence.size`                 | PersistentVolumeClaim storage size                                        | `8Gi`             |
+| `resources`                        | Container resource requests and limits in yaml                            | `{}`              |
+| `test.enabled`                     | Conditionally provision test resources                                    | `false`           |
```

Comment:
There are literally no charts that I can find that follow this pattern. I have looked at a couple of popular charts, and they just define tests without any gate. Not sure what is the concern with "creating extra resources".

Reply:
the test runner pod runs explicitly when `helm test` is executed.  However, a configmap was being created regardless of whether `helm test` was run.  This approach observes and controls test-resources.

configmap in question: https://github.com/kubernetes/charts/blob/master/stable/openldap/templates/tests/openldap-tests.yaml